lib/Models/Terria.ts

```diff
@@ -321,6 +321,11 @@ interface ConfigParameters {
 
   relatedMaps?: RelatedMap[];
 
+  /**
+   * Optional plugin configuration
+   */
+  pluginsConf?: Record<string, any>;
```

Comment:
I think the name of this should be either `plugins` (it'll basically always be used as `terria.configParameters.plugins`) or `pluginsConfig`, as config is our standard abbreviation of configuration

Reply:
Given the usage as `terria.configParameters.plugins` I like `plugins`, update coming

Reply:
Resolved in new commit